aws/policy/application-services.yaml

```
@@ -163,6 +163,14 @@ Statement:
       - logs:PutMetricFilter
     Resource:
       - 'arn:aws:logs:{{ aws_region }}:{{ aws_account_id }}:log-group:ansible-test*'
+  - Sid: DescribeEventRules
+    Effect: Allow
+    Action:
+      - events:DescribeRule
+      - events:CreateRule
+      - events:DeleteRule
```

```suggestion
      - events:DeleteRule
      - events:PutRule
      - events:ListTargetsByRule
      - events:PutTargets
```
The tests in 623 pass for me with these additional actions and the change I noted in that PR.

---

Changes done